lego/pytest_lego/plugin.py

```diff
@@ -1,54 +1,69 @@
+# type: ignore
+# pylint: skip-file
 """
 Implementation of the pytest-lego plugin.
 """
+from typing import Any
+
 import asyncio
 import functools
 import pytest
 import rpyc
 
 from . import component_factory
 
-MARK = "lego"
+MARK = 'lego'
 
 
-@pytest.fixture(scope="session")
-def components(request):
-    """
-    Provides the connection to the lego manager.
-    The 'components' fixture doesn't provide components but provides a way
+@pytest.fixture(scope='session')
+def _components(request) -> rpyc.Connection:
```

Comment:
Why you changed the fixture name? did you update the test_lego.py? You can change the function name but keep the fixture name by using `@pytest.fixture(name="bla")`

Reply:
Look above.
It is better to use the same name..